### PR TITLE
BEP update

### PR DESCRIPTION
**Pull request by franklin-feingold**
updated the BEPs with latest information and added a new column `Communication channels` - this is to explicitly identify where interested researchers can go 

addresses: https://github.com/bids-standard/bids-website/issues/80

the reason circle is failing is because it has not been initialized (simply turned on from https://github.com/bids-standard/bids-website/issues/81)

### CODE REVIEW

File: _pages/get_involved.md [hunk 1]
```diff
@@ -43,49 +44,49 @@ Proposals (BEPs), which are community-driven processes.
...
 | :---------------------------------------------------------------------------------------- | :----------------------------------------- | :------------------------------------------------ |
 | [BEP006](https://docs.google.com/document/d/1ArMZ9Y_quTKXC-jNXZksnedK2VHHoKP3HCeO5HPcgLE) | Electroencephalograpgy (EEG)               | Cyril Pernet, Stefan Appelhoff, Robert Oostenveld |
 | [BEP007](https://docs.google.com/document/d/1gs7USIVO1OApZs3IVo6TeXw9_tfzxhjM2-m--ex4g6o) | Hierarchical Event Descriptor (HED) Tags   | Chris Gorgolewski                                 |
 | [BEP008](https://docs.google.com/document/d/1FWex_kSPWVh_f4rKgd5rxJmxlboAPtQlmBc1gyZlRZM) | Magnetoencephalography (MEG)               | Guiomar Niso                                      |
 | [BEP010](https://docs.google.com/document/d/1qMUkoaXzRMlJuOcfTYNr3fTsrl4SewWjffjMD5Ew6GY) | intracranial Electroencephalograpgy (iEEG) | Chris Holdgraf, Dora Hermes                       |
+| [BEP015](https://docs.google.com/document/d/1WYOTXDB7GzlHoWqLjd45I3uGBgPxXddST-NTqBnroJE) | Mapping file                               | Eric Earl, Camille Maumet, and Vasudev Raguram    |
```
Comment (sappelhoff) [inline _pages/get_involved.md]: I missed BEP015 being merged into the BIDS specification, when did that happen? Could you give me a link to the PR, please?

**Comment by robertoostenveld** [comment]
I am not sure we should add a communication channels column here, as these things tend to change over time and it is difficult for us to maintain. Also, for some (e.g. [BEP005](https://docs.google.com/document/d/15tnn5F10KpgHypaQJNNGiNKsni9035GtDqJzWqkkP6c/edit#heading=h.prrzvwchfio6)) the instructions are quite elaborate (i.e. whom to contact to get invited for the bi-weekly Skype call). That is formatting-wise not so suitable for a cell in a table.

I would rather have a more elaborate (but consistent) heading of each BEP google doc, so that they list the communication channel(s). The google doc (or office365 doc, or the whatever system will be used in the future for this) is only one click away from the table.

So my proposal is that we update the [guidelines/recommendations](https://docs.google.com/document/d/1pWmEEY-1-WuwBPNy5tDAxVJYQ9Een4hZJM06tQZg8X4/edit#heading=h.kksepga0hf4o) for BEP leads, that the BEP SHOULD (not MUST) have:

- title (no version needed afaik)
- leads (2 or more leads preferably)
- communication channels
- status
- disclamer (i.e. "this is a draft ..." as in the box)
- and then the suggested changes

Also for the suggested changes, we could give recommendations, e.g. split it up in changes to the common spec, versus extensions that do not affect other modalities. Avoid repetition of the main spec. Describe which new [entities](https://bids-specification.readthedocs.io/en/stable/99-appendices/04-entity-table.html) are introduced and how to they intersect with other modalities ...

**Comment by effigies** [comment]
@franklin-feingold Can you merge/rebase master?

**Comment by franklin-feingold** [comment]
It appears this is a different direction than when this was openned. I am closing this in favor of simply updating with BEP updates with what I have. The additional information for BEP leads can be raised to the community and updated accordingly

**Pull request closed without merging**